phpunit.xml.dist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why leave this as ".dist?" Should this be the standard config for PHPUnit unit tests rather than an example?